sgtm/dioscuri/README.md

```diff
@@ -0,0 +1,126 @@
+# Dioscuri
+
+Get response from [Google Gemini 1.0 Pro and 1.5 Pro](https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/gemini) 
+in real time. This solution is useful for getting quick access to Gemini
+using sGTM, which enables a light-weight set-up and an easy integration
+with you website, cloud infrastucture, and third-party platforms.
+
+## Why Dioscuri?
+
+The [Dioscuri](https://en.wikipedia.org/wiki/Castor_and_Pollux) is the name 
+given to twin brothers Castor and Pollux. In Latin they are known as the Gemini.
+
+## Implementation
+Ready to start implementing this solution? You can follow the guide below which 
+outlines how Server Side Google Tag Manager (sGTM) can be used with Gemini.
+
+## Prerequisites
+
+-   Server Side Google Tag Manager
+-   Access to a Google Cloud project with billing enabled
+
+## Vertex AI API Setup
+
+1. In the Google Cloud console, on the project selector page, select a 
+   Google Cloud project.
+2. [Use this link to enable the Vertex AI API](https://pantheon.corp.google.com/apis/enableflow?apiid=aiplatform.googleapis.com&_ga=2.89950491.1404017710.1717753577-283650328.1717753577&project=gps-asa-uk-sgtm-demo-account)
```

Comment:
This is a corp link, and contains the project name. I think this is the link you need:

https://console.cloud.google.com/apis/enableflow?apiid=aiplatform.googleapis.com